src/test/scala/algolia/integration/RulesIntegrationTest.scala

```diff
@@ -62,6 +62,22 @@ class RulesIntegrationTest extends AlgoliaTest {
       }
     }
 
+    it("should not save a rule with empty objectID") {
+      val o = client.execute {
```

Comment:
why create an index as this test won't call the API?

Reply:
Because it's Monday 6pm and I'm already writing bad code. Removing it right now.